src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,25 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    width: 16px;
+    height: 16px;
+
+    &:not(:last-child) {
+      margin-right: 4px;
+    }
+  }
+
+  &--1 :first-child,
+  &--2 :nth-child(-n+2),
+  &--3 :nth-child(-n+3),
+  &--4 :nth-child(-n+4),
+  &--5 :nth-child(-n+5) {
+    background-image: url(../images/star-active.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+  }
```

Comment:
Do not repeat yourself

Also you can use loop

![2023-06-26_15h26_07](https://github.com/mate-academy/layout_product-cards/assets/61904995/28396eef-7f23-4183-8591-9051673919eb)

```suggestion
    background-image: url(../images/star-active.svg);
  }
```